chargebee/src/main/java/com/chargebee/android/billingservice/CBPurchase.kt

```diff
@@ -13,12 +13,18 @@ import com.chargebee.android.models.ResultHandler
 import com.chargebee.android.network.*
 import com.chargebee.android.resources.CatalogVersion
 import com.chargebee.android.resources.ReceiptResource
-import java.util.ArrayList
+
 object CBPurchase {
 
     var billingClientManager: BillingClientManager? = null
     val productIdList = arrayListOf<String>()
     private var customer : CBCustomer? = null
+    var inActivePurchases = false
+
+    enum class ProductType(val value: String) {
+        SUBS("subs"),
+        INAPP("inapp")
+    }
```

Comment:
What does this represent?

Reply:
If this is used only in `queryAllPurchaseHistory` can we instead make this internal with appropriate naming?

Reply:
Done